### ISSUE TITLE
Invalid python runtimee for execute-script resources

### ISSUE THREAD

**Issue by pdehlke** [aws-ia/cfn-ps-compliance-pci-fsbp-remediation#14]
All of the executeScript actions in the AWS::SSM::Document resources contained in aws-security-hub-pci-remediations-template1.template.yaml specify a runtime of python3.10. This is not a supported executeScrpipt runtime: https://docs.aws.amazon.com/systems-manager/latest/userguide/automation-action-executeScript.html

Was this a typo?

**Comment by pdehlke** [comment]
*bump*

Anyone reading these?

**Comment by Giles-W** [comment]
Where you ever able to work around your issue? I'm having issues using these templates as well and it's a touch frustrating that noone reads our posts.

**Comment by pdehlke** [comment]
No. You get what you pay for with OSS but one would have hoped that aws-ia might be responsive. 

@vsnyc you seem to be the only committer here. Anything?
